### PR TITLE
chore: update development docs to explain usage docs generation

### PR DESCRIPTION
**Pull request by benjamincharity**
ISSUES CLOSED: #1299

### CODE REVIEW

File: DEVELOPMENT.md [hunk 1]
```diff
@@ -297,6 +298,24 @@ machines.
...
+```
+// component:
+foo.component.ts
+// usage docs:
+foo.component.md
+```
+
+When the documentation published, this usage doc will be automatically consumed by the API docs generation.
+
+Whenever a usage document is edited, `yarn run docs:toc` should be run. This will automatically update the Table of Contents at the top of
```
Comment (shani-terminus) [inline DEVELOPMENT.md]: This is the wording that confused me.

To edit a doc, add headlines and appropriate content, then 'yarn run docs:toc` to update the table of contents.

File: DEVELOPMENT.md [hunk 2]
```diff
@@ -297,6 +298,25 @@ machines.
...
+```
+// component:
+foo.component.ts
+// usage docs:
+foo.component.md
+```
+
+When the documentation published, this usage doc will be automatically consumed by the API docs generation.
+
+When editing or creating usage documentation, add headlines and appropriate content, then run `yarn run docs:toc` to update (or generate)
```
Comment (benjamincharity) [inline DEVELOPMENT.md]: @shani-terminus how does the above look?

Comment (shani-terminus) [reply DEVELOPMENT.md]: Much better! Explicitly says first you edit, then run the command.

**Pull request merged**